_includes/avatar.html

```diff
@@ -1 +1 @@
-<img height="{{include.height}}" width="{{include.width}}" class="{{include.class}}" src="{% if post.avatar contains '://' %}{% elsif post.avatar %}{{ site.baseurl }}/hackergotchi/{% else %}{{ site.baseurl }}/hackergotchi/default.png{% endif %}{{ post.avatar }}"/>
+<img height="{{include.height}}" width="{{include.width}}" class="{{include.class}}" alt="the logo of {{post.author}}" src="{% if post.avatar contains '://' %}{% elsif post.avatar %}{{ site.baseurl }}/hackergotchi/{% else %}{{ site.baseurl }}/hackergotchi/default.png{% endif %}{{ post.avatar }}"/>
```

Comment:
I don't know if I would rely on `post` existing in this context

Reply:
I'll be sure to include an if statement in the final version. Thanks!